oss-cloud-frontend/src/App.vue

```diff
@@ -1,5 +1,10 @@
 <template>
   <v-app>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Material+Icons"
```

Comment:
I would prefer more to define font in this way (below the script element):

<style>
@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Material+Icons');
</style>

Reply:
Font is now in style tag.